### ISSUE TITLE
Nivu Week 3

### ISSUE THREAD

**Issue by nivupai** [noyah81/p2ducks#30]
Setting up profile page, backend of Sign-Up page, and link-in between the two.

**Setting up profile page and sign-up backend**

- [x] Create a base for the user's individual profile page 
- [x] Profile page will have customizable features
- [x] Fix the sign-up bottom and add to navbar
- [x] Blue print route from button on login to signup
- [x] Blue print route from button on login to profile page
- [x] Blue print route from button on signup to login page

TLDR:
* Allows the user to create profiles and become a part of the Hawker community
* User base for 'tweet' feature in terms of diplay and links to elvreything else

**Comment by sarahwxie** [comment]
Hey Nivu! I checked your code, and you've completed all these things & the UI looks really good! Everything is working without bugs which is really nice. Your ticket could be improved by bullet points, but other than that,  it looks good.

**Comment by nivupai** [comment]
> Hey Nivu! I checked your code, and you've completed all these things & the UI looks really good! Everything is working without bugs which is really nice. Your ticket could be improved by bullet points, but other than that, it looks good.

Thanks Sarah, I'll do that right now :)

**Comment by noyah81** [comment]
Reorganize the ticket to maybe add more details and structure so the the reader understands the purpose (like what the logging in feature is useful for in terms of the theme of our site).

**Comment by AkhileshLG** [comment]
The ticket looks very well put together. You have a good bit at the end that summarizes it and you have a list of everything that you worked on. You were also very specific about it. I don't really see anything that could be changed. Maybe you can put some future ideas?

**Comment by meghanakillada** [comment]
The ticket looks good and very detailed. I like how you incorporated checkboxes to show which components you completed and it makes it more organized. I think the ticket can be improved by adding links to code and runtime.

**Comment by noyah81** [comment]
Great job on this. I saw the nav bar update it looked way more organized. Specify what parts of the profile will be customizable.

**Comment by sarahwxie** [comment]
The profile page and backend to the sign up page is working really well! The color scheme looks nice on the website and the backend code has lots of comments to help me understand it.
